bigquery/main_test.go

```diff
@@ -431,7 +432,7 @@ func TestSendNotification(t *testing.T) {
 			BuildTriggerId: "1234",
 			Status:         0,
 		},
-		wantErr: true,
+		wantErr: false,
```

Comment:
Also check that no write attempt was made.

Reply:
Can you please add a `wantRow` bool or something and check for the added row? LMK if you need help structuring your test/interfaces to do this.